contracts/utils/ShortStringOps.sol

```diff
@@ -9,45 +9,8 @@ library ShortStringOps {
     using ShortStrings for *;
```

Comment:
optional: we can move ShortStringOps.sol library to the "lib" folder and delete the current folder where it is in

Reply:
1aa42a9